new_tutorials/04_distilling_a_reader.ipynb

```diff
@@ -0,0 +1,377 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Distil a Reader\n",
+    "\n",
+    "- **Level**: Advanced\n",
+    "- **Time to complete**: 45 minutes\n",
+    "- **Prerequisites**: Prepare the Colab environment (see links below).\n",
+    "- **Nodes Used**: `FARMReader`\n",
+    "- **Goal**: Distil the question answering capabilities of a larger Reader model into a smaller Reader model.\n",
+    "\n",
+    "Model distillation is the process of teaching a smaller model to imitate the performance of a larger, better trained model. By distilling one model into another, you end up with a more computationally efficient version of the original with only a slight trade-off in accuracy. In this tutorial, you will learn how to perform one form of model distillation on Reader models in Haystack. Model distillation is a complex topic and an active area of research so if you would like to learn more about it, we recommend looking at [Model Distillation](https://docs.haystack.deepset.ai/docs/model_distillation)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": false
+   },
+   "source": [
+    "## Preparing the Colab Environment\n",
+    "\n",
+    "- [Enable GPU Runtime in GPU](https://docs.haystack.deepset.ai/docs/enable-gpu-runtime-in-colab)\n",
+    "- [Check if GPU is Enabled](https://docs.haystack.deepset.ai/docs/check-if-gpu-is-enabled)\n",
+    "- [Set logging level to INFO](https://docs.haystack.deepset.ai/docs/set-the-logging-level)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Installing Haystack\n",
+    "\n",
+    "To start, let's install the latest release of Haystack with `pip`:"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%bash\n",
+    "\n",
+    "pip install --upgrade pip\n",
+    "pip install farm-haystack[colab]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Augmenting Training Data\n",
+    "\n",
+    "Having more training data is useful at all levels of model training. When performing intermediate layer distillation, additional data is beneficial, even if it is synthetically generated. Here we will be using the [`augment_squad.py` script](https://github.com/deepset-ai/haystack/blob/main/haystack/utils/augment_squad.py) to augment our dataset. It creates artifical copies of question answering samples by replacing randomly chosen words with words of similar meaning. This meaning similarity is determined by their vector representations in a GLoVe word embedding model."
```

Comment:
```suggestion
    "Having more training data is useful at all levels of model training. When performing intermediate layer distillation, additional data is beneficial, even if it is synthetically generated. In this tutorial, we'll be using the [`augment_squad.py` script](https://github.com/deepset-ai/haystack/blob/main/haystack/utils/augment_squad.py) to augment our dataset. It creates artificial copies of question answering samples by replacing randomly chosen words with words of similar meaning. This meaning similarity is determined by their vector representations in a GLoVe word embedding model."
```

Reply:
"Having more training data is useful at all levels of model training. When performing intermediate layer distillation, additional data is beneficial, even if it is synthetically generated." -> these sentences are rather inconcrete, too general. Can we change them to concrete explanations? e.g. When you're distilling a model, it's useful to have a lot of training data, even synthetically generated, because...